src/components/TaskOverlay.tsx

```
@@ -0,0 +1,395 @@
+
+            {/* Темы */}
+            {task.topics && task.topics.length > 0 && (
+              <div
+                className={`relative overflow-hidden rounded-2xl p-2.5 mb-2 ${overlayMounted ? 'task-overlay-block-in' : ''}`}
+                style={overlayMounted ? { animationDelay: '0.2s' } : undefined}
+              >
+                <h3
+                  className="text-sm font-tech font-bold flex items-center mb-2"
+                  style={{ color: '#e8f4f8', textShadow: '0 0 8px rgba(180, 220, 240, 0.3)' }}
+                >
+                  <svg className="w-4 h-4 mr-2" fill="currentColor" viewBox="0 0 20 20" style={{ color: 'rgba(180, 220, 240, 0.8)' }}>
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)